src/post/caption.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class Caption extends React.Component {
+    render() {
+        <p>
+            {this.props.caption}
+        </p>
+    }
+}
+
+export default Caption;
```

Comment:
Add a newline after the last line of code.

Reply:
👍